app/models/RankCounterItem.scala

```diff
@@ -0,0 +1,40 @@
+package models
```

Comment:
Quick question. Is there any reason that RankCounterItem is on root project? would be better locate this into s2couter_core project?